sdk/iothub/Azure.ResourceManager.IotHub/src/autorest.md

```diff
@@ -16,14 +16,84 @@ skip-csproj: true
 modelerfour:
   flatten-payloads: false
 
- 
+override-operation-name:
+  IotHubResource_CheckNameAvailability: CheckIotHubNameAvailability
+  ResourceProviderCommon_GetSubscriptionQuota: GetIotHubUserSubscriptionQuota
+
+rename-mapping:
+  IotHubNameAvailabilityInfo: IotHubNameAvailabilityResponse
```

Comment:
I think it should be `IotHubNameAvailabilityResult` to keep consistent with other RPs.